test/Modules/IexecPoco.test.ts

```diff
@@ -0,0 +1,40 @@
+// SPDX-FileCopyrightText: 2024 IEXEC BLOCKCHAIN TECH <contact@iex.ec>
+// SPDX-License-Identifier: Apache-2.0
+
+import { Address, BigInt, Bytes, ethereum } from '@graphprotocol/graph-ts';
+import { assert, describe, newTypedMockEventWithParams, test } from 'matchstick-as/assembly/index';
+import { DealSponsored } from '../../generated/Core/IexecInterfaceToken';
+import { handleDealSponsored } from '../../src/Modules';
+
+describe('IexecPoco', () => {
+    test('Should handle DealSponsored', () => {
+        // Define mock parameters
+        const dealId = Bytes.fromHexString('0x1234567890abcdef1234567890abcdef12345678');
```

Comment:
It should be 32 bytes
```suggestion
        const dealId = Bytes.fromHexString('0x1234567890abcdef1234567890abcdef1234567890abcdef1234567890abcdef');
```